sound/voice_data.h

```diff
@@ -35,7 +34,10 @@ const uint8_t spGOOD[] PROGMEM = {0x0A,0x28,0xCD,0x34,0x20,0xD9,0x1A,0x45,0x74,0
 const uint8_t spMORNING[] PROGMEM = {0xCE,0x08,0x52,0x2A,0x35,0x5D,0x39,0x53,0x29,0x5B,0xB7,0x0A,0x15,0x0C,0xEE,0x2A,0x42,0x56,0x66,0xD2,0x55,0x2E,0x37,0x2F,0xD9,0x45,0xB3,0xD3,0xC5,0xCA,0x6D,0x27,0xD5,0xEE,0x50,0xF5,0x50,0x94,0x14,0x77,0x2D,0xD8,0x5D,0x49,0x92,0xFD,0xB1,0x64,0x2F,0xA9,0x49,0x0C,0x93,0x4B,0xAD,0x19,0x17,0x3E,0x66,0x1E,0xF1,0xA2,0x5B,0x84,0xE2,0x29,0x8F,0x8B,0x72,0x10,0xB5,0xB1,0x2E,0x4B,0xD4,0x45,0x89,0x4A,0xEC,0x5C,0x95,0x14,0x2B,0x8A,0x9C,0x34,0x52,0x5D,0xBC,0xCC,0xB5,0x3B,0x49,0x69,0x89,0x87,0xC1,0x98,0x56,0x3A,0x21,0x2B,0x82,0x67,0xCC,0x5C,0x85,0xB5,0x4A,0x8A,0xF6,0x64,0xA9,0x96,0xC4,0x69,0x3C,0x52,0x81,0x58,0x1C,0x97,0xF6,0x0E,0x1B,0xCC,0x0D,0x42,0x32,0xAA,0x65,0x12,0x67,0xD4,0x6A,0x61,0x52,0xFC,0xFF};
 const uint8_t spPAUSE1[] PROGMEM = {0x00,0x00,0x00,0x00,0xFF,0x0F};
 const uint8_t spVOLTS[] PROGMEM = {0xe8,0xc8,0xaa,0xd3,0x4d,0x1d,0x23,0x3f,0xa8,0x76,0x51,0x73,0x4c,0x22,0xd4,0x4a,0x3d,0x4c,0xb2,0x89,0x2c,0x4f,0x8f,0x34,0xcb,0x25,0x0c,0xae,0x54,0xd3,0x63,0xb7,0x28,0xc4,0x76,0xee,0x96,0xdd,0xe2,0x98,0xc6,0xa1,0x8b,0x76,0x4b,0x7c,0x5c,0xa3,0x2c,0x29,0x2d,0x0d,0x7e,0x03,0xaa,0x24,0x97,0x2c,0xab,0x6d,0x94,0x31,0x95,0x8a,0x68,0x36,0xc1,0xd6,0xb2,0x2b,0x83,0xde,0x04,0x6b,0xdb,0xaa,0xf4,0x61,0x04,0xba,0x6d,0xb1,0xca,0xaa,0x31,0x71,0x77,0x82,0xca,0x24,0xd2,0xd5,0x54,0x0e,0xfa,0x8b,0xec,0x0c,0x55,0x27,0x08,0x58,0x36,0x82,0x00,0xd3,0x67,0x08,0x60,0xf9,0x0c,0x01,0x6c,0x5b,0xa1,0x80,0xe5,0x2a,0x15,0xb0,0x43,0x96,0x00,0x76,0xc8,0x22,0xc7,0x48,0xd3,0xea,0xea,0xe6,0x01};
-#else
+const uint8_t spAUTO[] PROGMEM = {0xac,0x4c,0x2d,0x8c,0x57,0x9d,0xb8,0x32,0xed,0x34,0x7e,0x8a,0x53,0xca,0x78,0xc2,0x75,0xb5,0x4a,0x51,0xe2,0x8d,0xd0,0xb5,0xb2,0x43,0x49,0xbd,0x4a,0x3a,0x6a,0x2f,0x25,0xd5,0x4c,0xcd,0x8a,0x33,0x94,0x9c,0xbc,0xa4,0x63,0xac,0x33,0x72,0xd2,0xe2,0xf6,0x32,0xc6,0xca,0x96,0x4a,0xc6,0x2a,0x17,0xa7,0x04,0x0e,0xde,0x2c,0xdb,0xbc,0x12,0xa4,0x78,0xb3,0x6c,0x0b,0x4a,0x90,0xe2,0x8f,0x72,0x23,0x2a,0x4e,0x5b,0xa7,0x63,0x8f,0x38,0x3b,0x1d,0xad,0xaa,0x5d,0xe2,0xec,0x75,0x68,0xb3,0x6a,0x88,0x63,0xb2,0xc6,0xf7,0xb2,0x21,0x0e,0xc5,0x13,0x37,0xca,0xaa,0xd8,0x0e,0x73,0x7a,0x0f,0x23,0x62,0xbd,0xd5,0xe8,0x2c,0x16,0x89,0xf5,0x16,0xe5,0xf1,0x84,0x28,0xe6,0xcd,0x15,0x47,0x55,0xa3,0x98,0xc5,0x30,0x0e,0x77,0x84,0xa2,0x20,0x23,0x58,0xcd,0x31,0x3c};
```

Comment:
So, these symbols only exists if you have FEMALE_TALKIE_VOICE defined?
I think that if there is no male version of the sound, it should go outside of the #ifdef.

Reply:
fixed